### DISCUSSION TITLE
Possible to map kebab-case class names to camelCase properties?

### DISCUSSION THREAD

**Discussion by dargue3** [Viijay-Kr/react-ts-css#170] [Q&A]
Hello and thank you for all the work on this extension. Truly makes CSS modules way better! I'm just wondering whether it's possible to have kebab CSS map to camel JS properties on the `classes` object. Something like this:

```scss
/* results.module.css */

.results-list {
  display: flex;
}
```
Which would map to:
```jsx
// SearchResults.tsx

import classes from './results.module.scss';

<div className={classes.resultsList}>
```

I've spent all my years using kebab-case classnames in CSS and camelCase in JS and really wishing I didn't need to give up one or the other 😅  Thanks again for the repo :)

**Comment by Viijay-Kr** [comment]
@dargue3 Thanks for asking this question. 

Firstly I'd like to understand how the kebab to camel case transformation work from the compiler perspective ? . Do you use any transformer to convert camel case to kebab case during compile time ? 

Secondly , I m not sure how this use case fits into the scope of this extension. I do get the benefit of having separate case conventions in css and js respectively , however I'm afraid how much could the VS code editing experience offer us to respect this practice. 

Even if I say that its possible to write a conversion program , I m not sure how much is it worth the effort.
